tests/page-objects/default/login/login.wdio.page.js

```diff
@@ -24,7 +24,8 @@ const login = async ({ username, password, createUser = false, locale, loadPage
   }
 
   if (loadPage) {
-    await commonPage.waitForLoaders();
+    const waitForPartialLoad = privacyPolicy || adminApp;
```

Comment:
What is the reason for making these specific (privacyPolicy, adminApp) instead of generic (isPartialLoad or waitForPartialLoad)? 
If there's another app or page, then we'll need to add another property.

Also, when reading the usage of this function:
 `await loginPage.login({ username: constants.USERNAME, password: constants.PASSWORD, adminApp: true });` 
The meaning of the `adminApp` flag wasn't straightforward and I thought that maybe it was force redirection to the admin page after login 😅

Reply:
The redirection to the admin app is in API. I understand the source of confusion. 
I didn't like something like partial load because we already had a 'loadPage' parameter that was used throughout. having two parameters 'loadpage' and 'partialload' seemed more confusing. are we loading or not? rather than being explicit about how it loads (partially), i was explicit about what i expected it to load. 

i don't expect the list of special page's to grow soon.

Reply:
> The redirection to the admin app is in API. I understand the source of confusion.

yes, my initial thought was a regular browser navigation